samples/Samples.Server/StartupWithRouting.cs

```diff
@@ -42,7 +42,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddGraphQL(builder => builder
                 .AddServer(true)
-                .AddDocumentExecuter<SubscriptionDocumentExecuter>()
+                .AddSubscriptionExecutionStrategy()
```

Comment:
This does need to be updated...

Reply:
Why?

Reply:
Compiler shows warning, so I changed.

Reply:
Well, that's correct.  So why didn't we merge this PR?

Reply:
#727

Reply:
Just to be clear, I'm agreeing with this change, not against it.  Just puzzled why the PR was closed without being merged.

Reply:
I did not want to resolve conflicts.

Reply:
I already resolved all conflicts